src/tools/crossgen2/ILCompiler.DependencyAnalysisFramework/DependencyAnalysisPerfEventSource.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+/// <summary>
+/// Performance events releated to the dependency graph.
+/// </summary>
+namespace ILCompiler.DependencyAnalysisFramework
+{
+    // The event IDs here must not collide with the ones used by ReadyToRunPerfEventSource.cs
+    [EventSource(Name = "Microsoft-ILCompiler-Perf")]
+    public class DependencyAnalysisPerfEventSource : EventSource
```

Comment:
Could you make this `internal`? We don't want this to be exposed out of the dependency analysis framework.

Reply:
Done.